Makefile

```diff
@@ -0,0 +1,29 @@
+##########------------------------------------------------------------##########
+##########- Build Latest Tkn binary ------------------------------------------##########
+##########------------------------------------------------------------##########
+
+STABLE_BUILD_URL := https://github.com/tektoncd/cli/releases/download/v${TKN_VERSION}/tkn_${TKN_VERSION}_Linux_x86_64.tar.gz
+BUILD_PATH=build/tkn/v${TKN_VERSION}
+
+.PHONY: build-tkn-binary-Linux-x86-64
```

Comment:
1. `build` is misleading, what about `download`?
2. `build-tkn-binary-Linux-x86-64` is quite long, I would remove OS and architecture as long as we have just one, ideally it could detect it automatically (in new PR)